x/stakingmiddleware/types/keys.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+// MinterKey is the key to use for the keeper store.
+var (
+	ParamsKey = []byte{0x01} // key for global staking middleware params in the keeper store
+)
+
+const (
+	// module name
+	ModuleName = "stakingmiddleware"
+
+	// StoreKey is the default store key for mint
+
+	StoreKey = "customstmiddleware" // not using the module name because of collisions with key "ibc"
```

Comment:
can you explain the comment here?

Reply:
so if i use word "staking" for store key there is conflict with a staking module from cosmos.
but yes. i took this kind of solution from other module so it need to change to `because of collisions with key "staking"`

Reply:
I would advise updating the comments to reflect the actual case here. Is there anywhere to read up on this issue? The only thing I can think of is collision based on the 'staking' prefix - but I cannot reason why that causes an issue when every staking key type has a byte key type prefix after the store name.

Reply:
I had the same vibes

Reply:
![FDFB309F-C464-41AF-995C-DA171794F727](https://github.com/ComposableFi/composable-cosmos/assets/102041955/8da7b1b3-bb09-4c80-803a-6fde35678da4)

@faddat @joe-bowman 
`Potential key collision between KVStores:stakingmiddleware - staking`

Reply:
Thank you. Not seen this before. Technically it is correct, in practice it wouldn’t happen because of the byte prefixes, but it is being correctly defensive

Reply:
actually i just checked that 
`StoreKey = "customstakingparams"`

so actually we can use `staking` but not as a prefix
thanks. will rename from "customstmiddleware" to "customstakingparams"